sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -3,9 +3,9 @@
 ### 4.0.0-beta.3 (2021-05-05)
 * Cosmos DB Spark 3.1.1 Connector Preview `4.0.0-beta.3` Release.
 #### Configuration Renames
-* Renamed data source name "cosmos.items" to "cosmos.oltp".
-* Renamed data source name "cosmos.changeFeed" to "cosmos.oltp.changeFeed".
-* Configuration renamed. See [Configuration-Reference](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos-spark_3-1_2-12/docs/configuration-reference.md) for more details.
+* Renamed data source name "cosmos.items" to "cosmos.oltp", See [PR](https://github.com/Azure/azure-sdk-for-java/pull/21121). 
```

Comment:
Are the cosmos.items and the other one class names, if yes they should be in code format. If they are path names they should be italic
 
Please see this page on docs rules for formatting tex: https://review.docs.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=master

Reply:
addressed.